pkg/controller/nodes/oneagent_dao.go

```diff
@@ -0,0 +1,53 @@
+package nodes
```

Comment:
oneagent_dao.go - what's dao?

Reply:
data access object; my thought was that it connects the data from the cluster (e.g. data about OneAgent instances) with the code. First I wanted to name it "_controller", but we already have a file with that name. Yet, "_dao" is not that good of a name either, so I am open for suggestions.